DEVELOPER-GUIDE.md

```diff
@@ -0,0 +1,82 @@
+# 1-Click App Development with Packer
+
+This guide outlines the essential files and considerations for creating a 1-Click app deployment using Packer.
+
+## Basic 1-Click Setup
+
+**Required Files:**
+
+* `example-1-click/template.json`: This file defines the packages to install, image build configuration, and installation locations.
+* `example-1-click/scripts/installer.sh`: This script performs the actual application installation on the Droplet.
+* `example-1-click/files/etc/update-motd.d/99-one-click`: This file contains basic instructions for users upon first login to the Droplet.
```

Comment:
Might be worth mentioning the _template directory here, since that's a skeleton meant to be a quick-start for any of these.

Reply:
Sure i'll add that

Reply:
Added